### ISSUE TITLE
Modal Component

### ISSUE THREAD

**Issue by marbuser** [blade-ui-kit/blade-ui-kit#44]
Would be cool to have alpinejs modal components. Especially since jetstream has some modals like this, and it’s definitely a component I dread making when doing new projects because it’s an inconvenience, so I think it would be perfect for Blade UI Kit. Could also get turbocharged by the end user once Laravel 8 comes out and forwarding attributes from component to component becomes a thing! 😊

**Comment by driesvints** [comment]
Yeah maybe although I'd think it'd be pretty shallow if it was a renderless component with just an alpine statemanager that would close the modal. Also, modals are pretty opinionated and there could be a hundred different ways on how people would tackle them.

We'd need something pretty concrete before continuing or a PR to get this started.
